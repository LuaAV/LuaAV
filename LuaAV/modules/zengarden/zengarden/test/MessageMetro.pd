#N canvas 894 221 263 455 10;
#X obj 21 19 loadbang;
#X obj 83 347 print;
#X obj 83 287 metro 500;
#X obj 83 318 int;
#X obj 119 317 + 1;
#X msg 68 211 stop;
#X obj 87 19 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 68 79 delay 3000;
#X msg 36 234 17;
#X msg 21 256 0;
#X obj 21 50 t b b b b b;
#X obj 135 242 1000;
#X obj 52 109 delay 6000;
#X obj 36 143 delay 9000;
#X obj 21 173 delay 12000;
#X connect 0 0 10 0;
#X connect 2 0 3 0;
#X connect 3 0 1 0;
#X connect 3 0 4 0;
#X connect 4 0 3 1;
#X connect 5 0 2 0;
#X connect 6 0 10 0;
#X connect 7 0 5 0;
#X connect 8 0 2 0;
#X connect 9 0 2 0;
#X connect 10 0 14 0;
#X connect 10 1 13 0;
#X connect 10 2 12 0;
#X connect 10 3 7 0;
#X connect 10 4 2 0;
#X connect 11 0 2 1;
#X connect 12 0 11 0;
#X connect 13 0 8 0;
#X connect 14 0 9 0;
