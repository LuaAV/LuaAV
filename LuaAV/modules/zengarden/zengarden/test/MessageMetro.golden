[@ 0.000ms] print: 0
[@ 500.000ms] print: 1
[@ 1000.000ms] print: 2
[@ 1500.000ms] print: 3
[@ 2000.000ms] print: 4
[@ 2500.000ms] print: 5
[@ 9000.000ms] print: 6
[@ 10000.000ms] print: 7
[@ 11000.000ms] print: 8
